swagger.yaml

```diff
@@ -3,11 +3,28 @@ info:
   description: 'Assisted installation'
   version: 1.0.0
   title: AssistedInstall
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
 host: api.openshift.com
 basePath: /api/assisted-install/v1
 tags:
   - name: Assisted installation
```

Comment:
Let's standardize the tags' names. camelCase/snake_case..

Reply:
Yes I agree, just didn't want to make "real" changes in this commit